### PR TITLE
issue-1072 Load recipients while draft is loading

### PR DESCRIPTION
**Pull request by michael-volynets**
#1072

### CODE REVIEW

**Comment by tomholub** [comment]
@michael-volynets when you install `tslint` extension in vscode (and restart vscode), it should show you red squiggles in `composer.ts` around line 517.
```
ERROR: /home/runner/flowcrypt-browser/extension/js/common/composer.ts[517, 11]: Promises must be handled appropriately
```

It's complaining because `parseRenderRecipients` is async and returns a promise. If you don't `await` it, it will silently drop errors (plus you may run into concurrency issues). It wants you to `await` it.

File: extension/js/common/composer.ts [hunk 1]
```diff
@@ -514,6 +514,7 @@ export class Composer {
           this.S.cached('input_to').focus();
           this.S.cached('input_to').val(headers.to.join(','));
           this.S.cached('input_text').focus();
+          this.parseRenderRecipients('harshRecipientErrs');
```
Comment (tomholub) [inline extension/js/common/composer.ts]: > ERROR: /home/runner/flowcrypt-browser/extension/js/common/composer.ts[517, 11]: Promises must be handled appropriately

This call is missing an `await`

**Comment by michael-volynets** [comment]
Hi, Tom,
I wanted to add there "await", but if I add await there, the composer is
loading more time than without it,
I mean without "await" everything works fine, composer loads faster, but
when it's loaded, we see loading icons on recipients list.

**Comment by tomholub** [comment]
Got it. If you don't want to wait for it to finish, you should add `.catch(Catch.reportErr)`. This way, if errors ever happen, they will be properly reported. This will make it stop complaining, too.

**Comment by michael-volynets** [comment]
Oh, Okay, Got It

**Pull request merged**